Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL := /bin/bash
+
+# The name of the executable (default is current directory name)
+TARGET := $(shell echo $${PWD\#\#*/})
+.DEFAULT_GOAL: $(TARGET)
+
+# These will be provided to the target
+VERSION := 1.0.0
+BUILD := `git rev-parse HEAD`
+
+# Use linker flags to provide version/build settings to the target
+LDFLAGS=-ldflags "-X=main.Version=$(VERSION) -X=main.Build=$(BUILD)"
```

Comment:
I guess this a fairly standard Makefile for Go, but is there any point in including these when we don't have a `cmd` binary? Not a showstopper, just asking.. 😉

Reply:
You are right, I copied it from a different project, so it can be omitted.

Reply:
@nilsmagnus Did you want to remove these lines? Then I think we can merge this.